packages/manager/apps/web/client/app/app.js

```diff
@@ -440,21 +440,6 @@ angular
       );
     },
   ])
-  .run([
```

Comment:
i think this part was buggy, calling login() multiple time for each URLS is wrong and useless